google/cloud/bigtable/table.cc

```diff
@@ -20,6 +20,8 @@
 #include <thread>
 #include <type_traits>
 
+#include "google/cloud/bigtable/internal/async_retry_unary_rpc.h"
```

Comment:
Please move this `#include` to the block of includes, note that the block will be sorted by `clang-format`.